README.md

```diff
@@ -24,9 +25,10 @@ This extension relies on an up-coming go-c8y-cli [extensions](https://github.com
 **Note:** Building `go-c8y-cli` requires go version ≥ 1.20.
 
 ### Shell (bash/zsh/fish)
+Checkout the [go-c8y-cli extension docs](https://feat-extensions-manager--goc8ycli.netlify.app/docs/tutorials/extensions/creating-an-extension/) for the latest version update
 
 ```sh
-go install github.com/reubenmiller/go-c8y-cli/v2/cmd/c8y@e25a3b0ba9b974346b1bf98c70300fb464a8add4
+go install github.com/reubenmiller/go-c8y-cli/v2/cmd/c8y@d6ebaddfd3182f657bf9ad37282f5ad1c25255b3
```

Comment:
I needed to use this to get the latest changes working:
go install github.com/reubenmiller/go-c8y-cli/v2/cmd/c8y@219a3eb7fb2f13ef86203e43758cfab00d700c12